TOOLING.md

```diff
@@ -0,0 +1,15 @@
+# Tooling
+
+This documents describes at a high level some the key tools used across `near-js`.
```

Comment:
Packages will eventually be scoped under `@near-js` so using that name as opposed to `near-api-js`.